src/backend/web/routes/posts.js

```diff
@@ -31,4 +32,8 @@ posts.get('/', async (req, res) => {
   );
 });
 
+posts.get('/count', async (req, res) => {
+  res.json(count.getPostsCount());
```

Comment:
The call to `getPostsCount` needs to be `await`ed, since it's async, and you'll need to wrap in a `try/catch` and give back a 500 if the redis call fails.

Reply:
Ok thanks for the feedback. I tried to fix this in the following commit.